java/client-apis/pom.xml

```diff
@@ -34,5 +34,9 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
```

Comment:
Is it necessary to introduce the new dependence here? The shaded jar is fat enough...

Reply:
fixed